### ISSUE TITLE
How to interprete the output of GBM for Survival analysis?

### ISSUE THREAD

**Issue by fritshermans** [harrysouthworth/gbm#51]
I'm interested in getting predictions expressed in expected lifetimes, survival curves etc. Could one please explain me what the output of `predict.gbm` is? Are these probably (log) partial hazards?
Your help is much appreciated.

**Comment by gregridgeway** [comment]
Yes, these are just like the predictions that would come out of coxph().



-------- Original message --------From: fritshermans <notifications@github.com> Date: 9/11/17  8:45 AM  (GMT-08:00) To: harrysouthworth/gbm <gbm@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [harrysouthworth/gbm] How to interprete the output of GBM for
  Survival analysis? (#51) 
I'm interested in getting predictions expressed in expected lifetimes, survival curves etc. Could one please explain me what the output of predict.gbm is? Are these probably (log) partial hazards?

Your help is much appreciated.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/harrysouthworth/gbm","title":"harrysouthworth/gbm","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/harrysouthworth/gbm"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"How to interprete the output of GBM for Survival analysis? (#51)"}],"action":{"name":"View Issue","url":"https://github.com/harrysouthworth/gbm/issues/51"}}}
